website/funkcje.js

```diff
@@ -144,7 +144,7 @@ function sprawdzNazwe() {
 
   var poleOut = doc.getElementById(poleOutId);
 
-  if (poleInValue.length >= 5) {
+  if (poleInValue.length >= 1) {
```

Comment:
search always limited by ajax. I've done this to get translated error message (from php instead of js)
I agree that it's better to return error asap, will try to change it